tests/test_errors.py

```diff
@@ -1,14 +1,19 @@
 import pytest
 
-from pydantic import VERSION, BaseModel, PydanticUserError
+from pydantic import BaseModel, PydanticUserError
 
 
 def test_user_error_url():
     with pytest.raises(PydanticUserError) as exc_info:
         BaseModel()
 
     # insert_assert(str(exc_info.value))
+    # assert str(exc_info.value) == (
+    #     'Pydantic models should inherit from BaseModel, BaseModel cannot be instantiated directly\n\n'
+    #     f'For further information visit https://errors.pydantic.dev/{VERSION}/u/base-model-instantiated'
```

Comment:
I kept this here commented just to make sure when it gets fixed later, the version doesn't end up hard-coded.